gluten-core/src/main/scala/org/apache/spark/util/SparkDirectoryUtil.scala

```diff
@@ -87,7 +91,7 @@ object SparkDirectoryUtil extends Logging {
     }
   }
 
-  def get(): SparkDirectoryUtil = synchronized {
+  def get(): SparkDirectoryUtil = {
     assert(INSTANCE != null, "Default instance of SparkDirectoryUtil was not set yet")
     INSTANCE
   }
```

Comment:
The get operation is not synchronized.

Reply:
Now, it is guarded with `INSTANCE_INITIALIZED`.